yolo_to_yolo/data_transformers.py

```diff
@@ -47,3 +47,139 @@ def __call__(self, input_data: YoloImageData) -> Iterable[YoloImageData]:
     def include(classname: str) -> bool:
         return classname != 'background' and len(classname) > 1
 
+class BBoxToCropTransformer(YoloDataTransformer):
+    """
+    Converts bounding box labels to cropped images
+    """
+    def __init__(self, min_size: tuple[int, int] = (0, 0), min_padding: int = 0):
+        self.min_size = min_size
+        self.min_padding = min_padding
+
+    def __call__(self, input_data: YoloImageData) -> Iterable[YoloImageData]:
+        img_height, img_width = input_data.image.shape[:2]
+        
+        # I hope this is right
+        shape_labels = [label for label in input_data.labels if "shape" in label.classname]
+        char_labels = [label for label in input_data.labels if "character" in label.classname]
+        
+        for shape_label in shape_labels:
+            if isinstance(shape_label.location, YoloBbox):
+                shape_bbox = shape_label.location
+                best_char_label = None
+                best_overlap = 0 # you can increase this up to 1 to make overlaps more "strict" (boxes will have to overlap more to be considered as a match)
```

Comment:
can you make this a parameter in the constructor so whoever uses this class doesn't have to edit this class's code to change the threshold? And move the comment to a docstring comment on the constructor.